devops/actions/cleanup/action.yml

```
@@ -0,0 +1,9 @@
+name: 'Cleanup'
+description: 'Cleanup work directory upon job finish'
+
+
+runs:
+  using: 'node12'
+  main: 'dummy.js'
+  post: 'cleanup.js'
+
```

```suggestion

runs:
  using: 'node12'
  main: 'dummy.js'
  post: 'cleanup.js'
```
Too many empty lines.

Out of curiosity: why not clean up in `main` and just put clean-up command after we done all other jobs?

---

> Out of curiosity: why not clean up in main and just put clean-up command after we done all other jobs?

When all steps are finished, GitHub starts executing post-processing jobs. If we cleanup before that, we remove some of actions' data, and post-processing fails. This is GHA-style RAII to avoid that problem.